src/components/AppProvider/utilities/withAppProvider/tests/withAppProvider.test.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
This never had coverage and was causing `codecov` to dip below 90

Reply:
You think it would be worth adding tests to ensure the `polaris` prop get passed to the wrapped component?